//
// RECURSIVESTRUCT.EPOCH
//
// Test case for ensuring that recursive structures are detected and handled as errors
// Issue #29
//

structure SelfRecurse : (integer(foo), SelfRecurse(bar), real(quux))

entrypoint : () -> ()
{
	debugwritestring("If this program executes, the test has FAILED!")
}
